course5/week2/ex02.cc

```diff
@@ -0,0 +1,89 @@
+#include <iostream>
+#include <cmath>
+#include <chrono>
+
+// ARS - 20170213
+// Cormacks algo from https://en.wikipedia.org/wiki/Fast_inverse_square_root
+// With slight modifications to C++-ifyusing dynamic c
+
+float Q_sqrt(float);
+
+float Q_sqrt( float number )
+{
+	long i;
```

Comment:
It is important that sizeof(long) == sizeof(float).
1. Use ```std::int32_t``` instead of ```long```
2. Use ```static_assert``` to check the condition at compile time